Controller/DebugManagerController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Leadin\SurvivalKitBundle\Controller;
+
+use Leadin\SurvivalKitBundle\Logging\LogContext;
+use Leadin\SurvivalKitBundle\Logging\Logger;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/debug-manager", name="survival_kit_debug_manager_")
+ */
+class DebugManagerController extends AbstractController
+{
+    /**
+     * @Route("/", name="manage", methods={"GET"})
+     */
+    public function manage(Request $request): Response
+    {
+        $aContexts = [];
+        $aConfig = $this->getConfig();
+        $sLogContextEnum = $this->getParameter('survival_kit.monolog.debug_manager.log_context_enum');
+        $aLogContexts = $sLogContextEnum::toArray();
+        \sort($aLogContexts);
+        foreach ($aLogContexts as $sContext) {
+            $aContexts[] = [
+                "name" => $sContext,
+                "expiration" => $aConfig[$sContext] ?? 0
+            ];
+        }
+
+        return $this->render('@SurvivalKit/debug_manager/manage.twig', [
+            "contexts" => $aContexts
+        ]);
+    }
+
+    /**
+     * @Route("/update-config/{sContext}/{sExpiration}", name="update_config", methods={"GET"})
+     */
+    public function updateConfig(string $sContext, string $sExpiration): Response
+    {
+        try {
+            $aConfig = $this->getConfig();
+            $aConfig[$sContext] = $sExpiration;
+
+            $sConfigPath = $this->getParameter('survival_kit.monolog.debug_manager.config');
+            $sDirPath = \dirname($sConfigPath);
+            if (!\is_dir($sDirPath)) {
```

Comment:
I found a way to do not force using Redis.
We use the [app](https://symfony.com/doc /current/cache.html#cache-configuration-with-frameworkbundle) cache, so each app can decide which adapter wants to use.

Reply:
It means that we have to set for each stack the "cache.adapter.system", adding then specific delivery step for each stack.
And I suppose we are going to choose : "cache.adapter.redis" and I suppose Symfony is going to use default Redis DB.

Reply:
> It means that we have to set for each stack the "cache.adapter.system", adding then specific delivery step for each stack.

App cache is enabled by default and uses `filesystem` adapter. So need to be configured only if want to use another adapter

> And I suppose we are going to choose : "cache.adapter.redis" and I suppose Symfony is going to use default Redis DB.

yep we choose redis, we can configure which Redis DB will be used

Reply:
of course we are going to use another cache system than filesystem otherwise better leave a flat file on the hard drive like it was before.

" we can configure which Redis DB will be used" seems very complexe deliveries steps for a very small added value, but why not ...

Reply:
freedom of choice is very BIG added value ;)

Reply:
cache configuration is quite simple and should be done in each stack independently from using it by ssk bundle